src/test/java/com/spotify/docker/client/messages/swarm/TaskTest.java

```diff
@@ -0,0 +1,49 @@
+/*-
+ * -\-\-
+ * docker-client
+ * --
+ * Copyright (C) 2016 - 2017 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.docker.client.messages.swarm;
+
+import static com.spotify.docker.FixtureUtil.fixture;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.spotify.docker.client.ObjectMapperProvider;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class TaskTest {
+
+  @Rule
+  public ExpectedException expectedException = ExpectedException.none();
+
+  private ObjectMapper objectMapper;
+
+  @Before
+  public void setUp() throws Exception {
+    objectMapper = ObjectMapperProvider.objectMapper();
```

Comment:
a minor style thing, but you can also just do

```java
private final ObjectMapper objectMapper = ObjectMapperProvider.objectMapper();
```

since junit re-instantiates the class for each `@Test`